src/services/db/GitFileSystemService.ts

```diff
@@ -433,9 +427,104 @@ export default class GitFileSystemService {
     })
   }
 
-  // TODO: Creates either directory or file
-  // ResourceDirectoryService used this to create a directory + file at the same time
-  create() {}
+  // Creates a file and the associated directory if it doesn't exist
+  create({
+    repoName,
+    userId,
+    content,
+    directoryName,
+    fileName,
+  }: {
```

Comment:
Should we standardise to passing function arguments as-is? Or should we pass as object instead like this?

Reply:
Hmm i think it might be clearer to pass as an object, but let's swap it to passing as in for consistency! Done in https://github.com/isomerpages/isomercms-backend/pull/892/commits/c6190b92eddd62f95ede7100a207565ed8a9e555